scripts/setup-database-and-test.sh

```
@@ -0,0 +1,383 @@
+async def collect_metrics():
+    service = SystemMetricsService()
+    start_time = time.time()
+    while time.time() - start_time < 30:
+        metrics = service.collect_metrics()
+        if metrics:
+            await service.store_metrics_async(metrics)
+        await asyncio.sleep(5)
+
+asyncio.run(collect_metrics())
+" &
+SYSTEM_METRICS_PID=$!
```

Shellcheck (0.10.0)

[warning] SYSTEM_METRICS_PID appears unused. Verify use (or export if used externally).

(SC2034)